pkg/checks/update_test.go

```diff
@@ -136,3 +136,35 @@ func createTestRepo(t *testing.T, dir, tag string) string {
 
 	return c.String()
 }
+
+func TestUpdateKeyExists(t *testing.T) {
```

Comment:
nit: might be nice to separate these test two test cases into separate `t.Run("test case name", ...)` instances to make it easier to review the test output